engine/src/main/java/org/destinationsol/drawable/components/Graphics.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.drawable.components;
+
+import org.destinationsol.drawable.GraphicsElement;
+import org.terasology.gestalt.entitysystem.component.Component;
+
+import java.util.ArrayList;
+
+/**
+ * Contains a list of the different elements that combine to visually represent an entity.
+ */
+public final class Graphics implements Component<Graphics> {
```

Comment:
Suggestion: Would `Renderable` be a better name? Sometimes it can be nice to name components after verbs since they're treats. Easier to say "That entity is renderable" vs "That entity is graphics" or even "that entity _has_ graphics" - plus I think "Graphics" is somewhat of an ambiguous term.

Inspiration as usual taken from Terasology, where admittedly it gets a bit more complicated since 3D and so on. So there just one of the things is `MeshComponent` but there are a bunch more and even variants of Mesh.